test/blackbox-tests/test-cases/merlin-tests/run.t

```diff
@@ -13,9 +13,9 @@
   B ../_build/default/lib/.foo.objs/public_cmi
   S $LIB_PREFIX/lib/bytes
   S $LIB_PREFIX/lib/findlib
+  S $TESTCASE_ROOT/lib
```

Comment:
Does this mean that we now use the absolute filename even for local libraries? I vaguely remember that we use to do that and changed it for some reason. Could you make sure we use `orig_src_dir` for external libraries only?

Reply:
Agree, sorry for bundling this meaningless change, fixed now.